fiftyone/core/utils.py

```diff
@@ -1488,6 +1529,23 @@ def __exit__(self, *args):
         logging.disable(logging.NOTSET)
 
 
+class add_sys_path(object):
```

Comment:
Where is this used? Also why is this class using `this_casing_style`?

Reply:
Something like this will need to be used by plugins that aren't Python packages (like GPT) until we find a better solution for local module imports.